jwala-services/src/main/java/com/cerner/jwala/service/configuration/service/AemServiceConfiguration.java

```diff
@@ -232,8 +234,8 @@ public ApplicationService getApplicationService(final JvmPersistenceService jvmP
 
     @Bean(name = "jvmControlService")
     public JvmControlService getJvmControlService(
-                                                  final JvmStateService jvmStateService,
-                                                  final HistoryFacadeService historyFacadeService) {
+            final JvmStateService jvmStateService,
```

Comment:
Since you modified the formatting already, wouldn't it be better if you formatted it this way ?

    public JvmControlService getJvmControlService(final JvmStateService jvmStateService, 
                                                  final HistoryFacadeService historyFacadeService) {
        ....
    }